src/wacks4shop/shift4shop.py

```diff
@@ -78,14 +80,6 @@ def determine_sales(self, timestamp: Optional[str]) -> list[Shift4ShopSale]:
         if timestamp:
             incomplete_orders_params["datestart"] = timestamp
         incomplete_orders_response = self._request_orders(incomplete_orders_params)
```

Comment:
can we use the helper method for searching orders instead of _request_orders? it might be nice to pass in some smaller limit < 300 but > 10 so it doesn't take so long to catch up